File: play-2.5.x/K-Modules/kmodules/K-Base/app/K/Aop/annotations/WithPersistLog.java
package K.Aop.annotations;

import java.lang.annotation.Retention;
import java.lang.annotation.Target;

import static java.lang.annotation.ElementType.TYPE;
import static java.lang.annotation.RetentionPolicy.RUNTIME;


/**
 * Created by kk on 14/11/28.
 */
@Target(TYPE)
@Retention(RUNTIME)
public @interface WithPersistLog {
}

File: play-kotlin-2.5.x/PlayKotlin/kmodules/K-Base/app/K/Aop/annotations/WithPersistLog.kt
package K.Aop.annotations

import java.lang.annotation.Retention

import java.lang.annotation.ElementType.TYPE
import java.lang.annotation.RetentionPolicy.RUNTIME


/**
 * Created by kk on 14/11/28.
 */
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
@Retention(RUNTIME)
annotation class WithPersistLog
